cfme/intelligence/reports/dashboards.py

```diff
@@ -127,8 +128,9 @@ def delete(self, cancel=False):
         sel.handle_alert(cancel)
         flash.assert_no_errors()
 
-    def reset_widgets(self, cancel=False):
-        sel.force_navigate("dashboard")
-        sel.click(self.reset_button, wait_ajax=False)
+    @navigate('dashboard')
+    @classmethod
```

Comment:
Because this method is used in different places and creating a dashboard object doesn't make a sense in widgets() fixture.